packages/vue-test-workspace/vue-tsc/non-strict-template/#3629/main.vue

```diff
@@ -0,0 +1,4 @@
+<script setup lang="ts">
+// @ts-expect-error
```

Comment:
With `@ts-expect-error` this test will pass both with and without the fix so it's missing the point.

Reply:
Ugh, I'm not sure how to fix this since `foo` doesn't exist on `defineProps()`. Any ideas?